projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -11,24 +11,31 @@ import { debounceTime, takeUntil } from 'rxjs/operators';
 
 import { Keys } from '../../../utils/enums/keys.enum';
 
+const actionableItemSelectors = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled])',
+  'button:not([disabled])',
+  'select:not([disabled])',
+  'textarea:not([disabled])',
+  'iframe',
+  'object',
+  'embed',
+  '*[contenteditable=true]',
+  '[role=button]:not([disabled])',
+];
+
 export function getTabableItems(el: HTMLElement) {
-  const tabableSelector = [
-    'a[href]',
-    'area[href]',
-    'input:not([disabled])',
-    'button:not([disabled])',
-    'select:not([disabled])',
-    'textarea:not([disabled])',
-    'iframe',
-    'object',
-    'embed',
-    '*[tabindex]:not([disabled])',
-    '*[contenteditable=true]',
-    '[role=button]:not([disabled])',
-  ].join(',');
+  const tabableItemSelectors = [...actionableItemSelectors, '*[tabindex="0"]:not([disabled])'];
```

Comment:
```suggestion
  const tabableItemSelectors = [...actionableItemSelectors, '[tabindex="0"]:not([disabled])'];
```

This `*` can be removed as well.